e2e/components/AvatarStack.test.ts

```diff
@@ -3,309 +3,371 @@ import {visit} from '../test-helpers/storybook'
 import {themes} from '../test-helpers/themes'
 
 test.describe('AvatarStack', () => {
-  test.describe('Default', () => {
-    for (const theme of themes) {
-      test.describe(theme, () => {
-        test('default @vrt', async ({page}) => {
-          await visit(page, {
-            id: 'components-avatarstack--default',
-            globals: {
-              colorScheme: theme,
-            },
-          })
+  for (const featureFlagOn of [true, false]) {
```

Comment:
One question, how do you feel about using this format for defining tests and running them:

https://github.com/primer/react/blob/main/e2e/components/Link.test.ts#L5

Also wanted to ask if you knew how to filter tests by describe block names? I wasn't sure how to do it 🤔 It's why I've been using `styled-components` in the name so that I could filter by that in the test name when running playwright locally